### ISSUE TITLE
Small Error with skill effect in "full_skills.js"

### ISSUE THREAD

**Issue by TubePowered** [aqiu384/aqiu384.github.io#2]
Hey, this is most certainly a minor issue, but I figured I should bring to light that there is one error (that I currently know of; I might scour the whole thing if you'd like me to) with a skill's description/effect. It's the passive skill "Angelic Grace." It's currently noted as "Double evasion rate against all attacks except Bless/Curse/Almighty," but I've just confirmed that the skill actually applies this evasion boost to both Bless and Curse moves, as well. No word on insta-kill effectiveness or anything, as it's not included in its in-game description. I just sort of noticed it while tinkering around with fusing/itemizing stuff in the VR and trying to remind myself just what the skill did as Raphael can turn into its skill card.

Anyway, tl;dr is that Angelic Grace is incorrect and should read as such: "Double evasion rate against all attacks except Almighty," and I'd be willing to look around a bit for any other errors in case you'd like me to. Hopefully this isn't too trivial a bug! I've mentioned it in part because it's referenced in other projects. ...And I hope this is helpful to you!

(I can take a photo of the in-game description to show as verification via cellphone since screenshots are disabled in P5 by Atlus)

**Comment by aqiu384** [comment]
Confirmed the description in-game. Fixed as of 948f294d3deac29d65eab1ac374100411eecf9c0
